package.json

```diff
@@ -33,6 +33,7 @@
     "not OperaMobile > 0"
   ],
   "dependencies": {
+    "@microsoft/fetch-event-source": "^2.0.1",
```

Comment:
Please don't add this to the overall `package.json`, but to the package(s) where this lib is needed.